guides/client-json-configuration-example.md

```diff
@@ -0,0 +1,206 @@
+# Client JSON Configuration Example
+
+This document provides an example JSON configuration that can be used with `Client.fromConfig()` across the Hiero SDKs. The configuration defines all necessary parameters to connect to and interact with a Hedera network.
+
+## Example Configuration
+
+```json
+{
+  "network": {
+    "35.242.233.154:50211": "0.0.3",
+    "35.240.118.96:50211": "0.0.4",
+    "34.127.45.23:50211": "0.0.5"
+  },
+  "mirrorNetwork": [
+    "hcs.testnet.mirrornode.hedera.com:443"
+  ],
+  "shard": 0,
+  "realm": 0,
+  "grpcDeadline": 10000,
+  "requestTimeout": 120000
+}
+```
+
+## Configuration Parameters
+
+### Network Configuration
+
+- **`network`** (object, optional): Maps consensus node addresses to their corresponding account IDs
```

Comment:
It does not make sense to define a config without any consensus node. Therefore for the current SDK the network should not be optional. Might be different in V3 of SDKs (see https://github.com/hiero-ledger/sdk-collaboration-hub/blob/main/v3-sandbox/prototype-api/config.md)